microsoft/PowerToys
# [Settings] Add VC142 CRT MergeModules to installer

## Summary of the Pull Request
Added the Visual C++ Redistributables for Visual Studio 2019 by including the Merge Modules for the CRT in the Wix installer.

 
## References


## PR Checklist
* [x] Applies to #2851


## Detailed Description of the Pull Request / Additional comments
* This solves the Settings V2 crash when trying to open the Settings page in a Windows 19h1 installation without the Redistributable package.
* The MergeModules included in binary form come from my Visual Studio 2019 installation, after downloading the C++ 2019 Redistributable MSMs Individual component.


## Validation Steps Performed
1. Start a brand new Windows 19h1 VM.
2. Build a MSI installer for PowerToys.
2. Install and lauch PowerToys on the VM.
3. Double click the PowerToys icon.
3. The Settings window appears.
